test/unit/test-helpers.js

```diff
@@ -23,7 +23,9 @@ const TestHelpers = {
 
     const player = new Player(videoTag, playerOptions);
 
-    player.middleware_ = [player.tech_];
+    if (addTechAsMiddleware) {
```

Comment:
I'm not sure why this is necessary

Reply:
When using the HTML5 tech and not TechFaker, the tech shouldn't be added as a middleware. Adding the tech as middleware makes things happen in a different order then the way that they actually happens outside of tests.